app/utils/prepareAnswer.js

```diff
@@ -17,17 +17,7 @@ function prepareStatsCompactAnswer(body, index, nameState) {
     if (parseInt(statedata['deltaconfirmed']) > 0) {
         data += '<i> (+' + numberWithIndianCommas(statedata['deltaconfirmed']) + ')</i>'
     }
-    data += "\n\u{1F7E0} Active: " + numberWithIndianCommas(statedata['active'])
- 
-        let deltaActive = parseInt(statedata['deltaconfirmed']) - parseInt(statedata['deltarecovered']) - parseInt(statedata['deltadeaths'])
-       //change in active number will not be displayed if it is zero.
-        if (deltaActive > 0)
-            data += '<i> (+' + numberWithIndianCommas(deltaActive) + ')</i>'
-        else if (deltaActive < 0) {
-            //required to display change in active numbers correctly if active number decreases
-            deltaActive = deltaActive + (-2) * deltaActive
-            data += '<i> (-' + numberWithIndianCommas(deltaActive) + ')</i>'
-        }
+    data += "\n\u{1F7E0} Active: " + numberWithIndianCommas(statedata['active']
```

Comment:
(statedata['active'])
missing the closing bracket.

Reply:
whops sorry, on it.

Reply:
done 👍